commands/credentials.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function * run (context, heroku) {
+  const fetcher = require('../lib/fetcher')(heroku)
+
+  const {app, args, flags} = context
+
+  let showCredentials = co.wrap(function * () {
+    let db = yield fetcher.database(app, args.database)
+    cli.log(`Connection info string:
+   "dbname=${db.database} host=${db.host} port=${db.port || 5432} user=${db.user} password=${db.password} sslmode=require"
+Connection URL:
+   ${db.url.href}`)
+  })
+
+  let reset = co.wrap(function * () {
+    const host = require('../lib/host')
+    let db = yield fetcher.addon(app, args.database)
+    yield cli.action(`Resetting credentials on ${cli.color.addon(db.name)}`, co(function * () {
+      yield heroku.post(`/client/v11/databases/${db.name}/credentials_rotation`, {host: host(db)})
+    }))
+  })
+
+  if (flags.reset) {
+    yield reset()
+  } else {
+    yield showCredentials()
+  }
+}
+
+module.exports = {
+  topic: 'pg',
+  command: 'credentials',
+  description: 'display the database credentials',
```

Comment:
As long as we're here, can you `s/display/manage/`? Right now it's not obvious from the topic-level help that this can reset credentials.